pkg/sql/ttl/ttljob/ttljob.go

```diff
@@ -211,6 +219,14 @@ func (t rowLevelTTLResumer) Resume(ctx context.Context, execCtx interface{}) err
 	p := execCtx.(sql.JobExecContext)
 	db := p.ExecCfg().DB
 	descs := p.ExtendedEvalContext().Descs
+
+	if enabled := jobEnabled.Get(p.ExecCfg().SV()); !enabled {
```

Comment:
where do background job errors like this show up? somewhere in the console?

Reply:
`SHOW JOBS` displays these